kratos/python_scripts/vtk_output_process.py

```diff
@@ -8,9 +8,9 @@ def Factory(settings, model):
     return VtkOutputProcess(model, settings["Parameters"])
 
 
-class VtkOutputProcess(KratosMultiphysics.Process):
+class VtkOutputProcess(KratosMultiphysics.OutputProcess):
```

Comment:
Shouldn't we do so with the GiD output process?

Reply:
This is only an example to ensure the usage

In fact there are more places where we should use it but I would update in separate PRs